src/main/webapp/dev/js/common/questionContrib.es6

```diff
@@ -23,6 +25,9 @@ function setDefaultContribQnVisibilityIfNeeded(questionNum) {
     $currentQuestionTable.find('input.visibilityCheckbox')
                          .filter('.recipientCheckbox')
                          .filter('[value="INSTRUCTORS"],[value="RECEIVER"]').prop('checked', true);
+    // Set dropdown button text
+    setVisibilityDropdownMenuText('Please select a visibility option <span class="caret"></span>',
```

Comment:
I'm a little hesitant to hardcode the message here, as it comes from a Java constant (what if the Java constant is updated next time, and not this)? However I'm not sure if there is a better solution though. Do you have any ideas?